handlers/guildHandlers.js

```diff
@@ -1,20 +1,22 @@
 function createOnMuteRole(guild) {
-  if (
-    guild.available &&
-    !guild.roles.cache.some((role) => role.name === 'On Mute')
-  ) {
-    guild.roles
-      .create({
-        name: 'On Mute',
-        color: 'DARK_BUT_NOT_BLACK',
-        permissions: [],
-      })
-      .then(console.log)
-      .catch(console.error);
+  if (guild.available) {
+    if (!guild.roles.cache.some((role) => role.name === 'On Mute')) {
+      guild.roles
+        .create({
+          name: 'On Mute',
+          color: 'DARK_BUT_NOT_BLACK',
+          permissions: [],
+        })
+        .then(console.log)
```

Comment:
I know this was here before, but it doesn't seem to be doing anything. Do we need it?

Reply:
It does a console.log of the newly created role object so it does provide some feedback on the success of role creation, mostly in development.

Reply:
Interesting. I would have thought you need to pass the role object into the `console.log` function for it to output it. Does it get implicitly passed in somehow?

Reply:
I understand it as console.log is initially not invoked but passed as the actual callback function. Which upon a successful promise resolution is invoked with the returned role object as the single parameter.